### ISSUE TITLE
Where is the color set for the codebox language text ?

### ISSUE THREAD

**Issue by compwiz32** [PKM-er/Blue-Topaz_Obsidian-css#458]
I have edited all my codebox color settings to make the colors be more readable. However, I cannot find where the code box language color is set. This is also the color for the code brackets as well.

Could someone please point me to where that super dark grey color is coming from? 

See the screencap - the red arrows point toward the items affected by the dark color.... 

![image](https://user-images.githubusercontent.com/16006392/204125697-c751f5d0-e48a-4081-b217-0648e02478da.png)

**Comment by Ali-C-Ila** [comment]
Not a direct answer, but you can try the **Settings Search** plugin. It allows you to search within the settings menu, which may shed some light on your issue.

**Comment by Ali-C-Ila** [comment]
Hmm I also don't like the color because of its low contrast, so I did some experiments. Here's what I found:
1. The color codes for that gray is #666666
2. #666666 remains the same across different themes, light/dark mode.
3. #666666 does not appear anywhere in settings.

TLDR:  I guess that cannot be customized.

**Comment by cumany** [comment]
the code box language color:
![image](https://user-images.githubusercontent.com/42957010/204130294-baaf75a2-c699-4c34-b607-989d92691622.png)

Code bracket color:
Find the following customization item to set
![image](https://user-images.githubusercontent.com/42957010/204127067-8b7b8d3e-c012-440b-b03d-140b7c464284.png)

**Comment by compwiz32** [comment]
Thank you! it feels like I had checked the color options about 30x and have never seen "Code Block punctuation Color" and "Code Block language Color". Then you mention and there they are right in front of me...

Also thanks for the Search settings plugin tip. Such a helpful plugin.

**Comment by cumany** [comment]
![image](https://user-images.githubusercontent.com/42957010/204150107-92d5f184-43fc-4026-8946-3d14124d7226.png)

Custom Codebox Syntax Highlighting  Colors
